.github/workflows/main.yml

```
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
```

This should be `master` (default branch name in the repos of the onosproject organization)
```suggestion
      - master
```

---

Done.